lib/pharos/transport/command/result.rb

```diff
@@ -115,22 +115,16 @@ def debug_exit(exit_status)
 
         private
 
-        if !ENV['DEBUG'].to_s.empty?
-          def debug(&block)
-            instance_exec(&block)
-          end
-        else
-          def debug(&_block)
-            nil
-          end
+        def debug(&block)
```

Comment:
The previous implementation required the ENV to be set already when the class was being loaded, which made `--debug` not enable the SSH debug correctly.